README.md

```diff
@@ -26,14 +26,32 @@ This specification defines formats to pass trace context information across syst
 to share this with the community so that various tracing and diagnostics products can operate
 together.
 
-## References
+## Reference Implementations
 
-- Cloud Events define [distributed tracing
-  extension](https://github.com/cloudevents/spec/blob/master/extensions/distributed-tracing.md)
+There are few open source implementations of this trace context specification
+available.
 
-## Reference Implementations
+A simplistic regex-based implementation can be found in the `test` folder. This
+implementation has 100% compliance to the test suite.
+
+[Open Census](https://opencensus.io) has implementations for the following
+languages:
+
+1. [C#](https://github.com/census-instrumentation/opencensus-csharp/blob/4a8ddf6727eafda97a06c7c30d8a4fc2ec8b8e2f/src/OpenCensus/Trace/Propagation/TraceContextFormat.cs)
+2. [Go](https://github.com/census-instrumentation/opencensus-go/blob/ae11cd04b7789fa938bb4f0e696fd6bd76463fa4/plugin/ochttp/propagation/tracecontext/propagation.go)
+3. [Java](https://github.com/census-instrumentation/opencensus-java/blob/e5e9d9224a1c9c5ee981981cf29e86662aef08c6/impl_core/src/main/java/io/opencensus/implcore/trace/propagation/TraceContextFormat.java)
+4. [Node.js](https://github.com/census-instrumentation/opencensus-node/blob/fa97a9b6f19b97e1038ffa9e1be4b407f3844df2/packages/opencensus-propagation-tracecontext/src/tracecontext-format.ts)
+5. [Python](https://github.com/census-instrumentation/opencensus-python/blob/2aef803e4a786fe0ffb14b168a8458283ccd72a0/opencensus/trace/propagation/trace_context_http_header_format.py)
+6. [Ruby](https://github.com/census-instrumentation/opencensus-ruby/blob/8cb9771b218e440e825c99981ea405d40f735926/lib/opencensus/trace/formatters/trace_context.rb)
+
+.NET Framework will ship trace context specification support in the upcoming
+version. See
+[Activity](https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs)
+for implementation details.
+
+Elastic has [node.js implementation](https://github.com/elastic/node-traceparent).
 
-TODO: add link here
+LightStep has [Go implementation](https://github.com/lightstep/tracecontext.go).
```

Comment:
Why are there TODOs in there. Feels incomplete for a merge. If you want a way to add vendors we should have a note and an issue where people can comment that they want their implemenation to be added.

Reply:
TODO was removed - see it has `-` sign in front of it and marked as red

Reply:
I was listing open source implementations that I found with github search. If you have one with DynaTrace - happy to add it

Reply:
>mentioning opencensus once

my goal was to highlight implementations in various languages. It would be helpful for people who wants to start using it to know where to find implementations for their language.

BTW, I haven't included PHP. Perhaps need to do it.

Reply:
SGTM